crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -678,53 +682,106 @@ impl TryFrom<TransactionEffects> for SuiTransactionBlockEffects {
     }
 }
 
+fn owned_objref_string(obj: &OwnedObjectRef) -> String {
+    format!(
+        " ┌──\n | ID: {} \n | Owner: {} \n | Version: {} \n | Digest: {}\n └──",
```

Comment:
nit: You can copy the vertical line-drawing characters that `tabled` uses to get lines without gaps in them.